tests/testthat/default_linter_testcode.R

```diff
@@ -25,7 +25,6 @@ g <- function(x) {
 # object_name
 # object_usage
 # open_curly
-# return
```

Comment:
oh! in some commit I added an explicit return here. Must not have survived. I see someone (possibly me?) added a `return_linter()` test case to this file in another commit.